### DISCUSSION TITLE
Reinstalled Win10, unable to get SD WebUI or Forge to work.

### DISCUSSION THREAD

**Discussion by Hunt3rseeker** [lllyasviel/stable-diffusion-webui-forge#687] [Q&A] [answered]
Right, so, I recently made a fresh re install of my windows 10. When I backed up my files before that, I thought that I should save the models, loras, wildcards etc, and skip the rest since I can just download it again.

Well, I've been trying all day today and yesterday to get this to work, with no success. I have no fucking idea why it worked flawlessly before, and not at all now. I'm on the brink of just giving up, and so, now I am here, asking for guidance.

I'll try to explain what I've done as clearly as I can, sorry for any bad grammar, it's a side effect of being born in Europe.

I ran SD Forge before I reinstalled windows, so I thought I'll just get that again. It worked without errors, created images fast, and I liked the extra settings it had. Downloaded from [here](https://github.com/lllyasviel/stable-diffusion-webui-forge), the one that's called "One-click package". Installed it on C:\, and started it up. I move all the models and extensions to the new folders, and refresh--everything now show's up in the UI. But when I generate images, next to nothing happened. Only thing was that the generate button showed skip and interrupt. Otherwise, nothing, not even in cmd.

Other buttons where unresponsive, like I was trying to refresh the extension page, but it just said loading. I thought that maybe I did something wrong in the installation process, so I did it again and deleted the current one. Nope, same thing. At this point I say fuck it, just get the old SD-WebUI instead.

Now, this was actually pretty frustrating to install. I was looking around for guides (I'm not a programmer or have any background of programming) to make sure I was doing it right. But MANY guides show entirely different ways to install SD-WebUI. Some guides skip cuda, ffmpeg or microsoft visual studio C++ compile tools, while others say that they are crucial for the installation to work. And others just install python 3.10 and Git and do nothing more. Extremely confusing for me as a programming noob. I just settled to follow [SECourses video - Essential AI Tools and Libraries: A Guide to Python, Git, C++ Compile Tools, FFmpeg, CUDA, PyTorch](https://www.youtube.com/watch?v=-NjNy7afOQ0) since it seemed to go more into detail on how to do this. In short, I installed all of the programs shown in the link above, and [SD WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui) at "Installation on Windows 10/11 with NVidia-GPUs using release package".

Followed every single step, and yet I am met with errors such as:

    Failed to create model quickly; will retry using slow method.
    loading stable diffusion model: OSError
    Traceback (most recent call last):
      File "C:\Python310\lib\threading.py", line 973, in _bootstrap
        self._bootstrap_inner()
      File "C:\Python310\lib\threading.py", line 1016, in _bootstrap_inner
        self.run()
      File "C:\stable-diffusion-webui\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 807, in run
        result = context.run(func, *args)
      File "C:\stable-diffusion-webui\venv\lib\site-packages\gradio\utils.py", line 707, in wrapper
        response = f(*args, **kwargs)
      File "C:\stable-diffusion-webui\modules\ui.py", line 1154, in <lambda>
        update_image_cfg_scale_visibility = lambda: gr.update(visible=shared.sd_model and shared.sd_model.cond_stage_key == "edit")
      File "C:\stable-diffusion-webui\modules\shared_items.py", line 175, in sd_model
        return modules.sd_models.model_data.get_sd_model()
      File "C:\stable-diffusion-webui\modules\sd_models.py", line 620, in get_sd_model
        load_model()
      File "C:\stable-diffusion-webui\modules\sd_models.py", line 732, in load_model
        sd_model = instantiate_from_config(sd_config.model)
      File "C:\stable-diffusion-webui\repositories\stable-diffusion-stability-ai\ldm\util.py", line 89, in instantiate_from_config
        return get_obj_from_str(config["target"])(**config.get("params", dict()))
      File "C:\stable-diffusion-webui\repositories\stable-diffusion-stability-ai\ldm\models\diffusion\ddpm.py", line 563, in __init__
        self.instantiate_cond_stage(cond_stage_config)
      File "C:\stable-diffusion-webui\repositories\stable-diffusion-stability-ai\ldm\models\diffusion\ddpm.py", line 630, in instantiate_cond_stage
        model = instantiate_from_config(config)
      File "C:\stable-diffusion-webui\repositories\stable-diffusion-stability-ai\ldm\util.py", line 89, in instantiate_from_config
        return get_obj_from_str(config["target"])(**config.get("params", dict()))
      File "C:\stable-diffusion-webui\repositories\stable-diffusion-stability-ai\ldm\modules\encoders\modules.py", line 103, in __init__
        self.tokenizer = CLIPTokenizer.from_pretrained(version)
      File "C:\stable-diffusion-webui\venv\lib\site-packages\transformers\tokenization_utils_base.py", line 1809, in from_pretrained
        raise EnvironmentError(
    OSError: Can't load tokenizer for 'openai/clip-vit-large-patch14'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure 'openai/clip-vit-large-patch14' is the correct path to a directory containing all relevant files for a CLIPTokenizer tokenizer.
    
    
    Stable diffusion model failed to load

Which leaves me puzzled because that model is in the model folder "C:\stable-diffusion-webui\models\Stable-diffusion\v1-5-pruned-emaonly.safetensors". It shows up in the drop down menu, but it never loads when I choose it.

So I've narrowed the problems down to that either, I must be a fucking idiot and I'm missing crucial steps on how to install SD WebUI and/or Forge, or something has happened with my computer when I reinstalled Windows. Not sure what it might be. 

I greatly appreciate any help or guidance. I think personally what might help me the most is a **FOOL-PROOF** step-by-step guide, starting from zero. But if anyone thinks otherwise, please, be my guest! I just wanna make silly pictures :'(

**Comment by Hunt3rseeker** [comment]
Got it to work somehow. Just reinstalled it another time, and it just worked. Shrug

**Reply by Hasiga** [reply]
tengo el mismo problema, que cosa reinstalaste? 

I have the same problem, what did you reinstall?

**Reply by Hunt3rseeker** [reply, marked as answer]
I believe I was referring to reinstalling Forge, that is deleting the root folder. Before that, I remember uninstalling/reinstalling python and git, but that was before I reinstalled Forge a bunch of times. Honestly, I was flabbergasted that it just worked suddenly. I really felt like I was doing the same process, and then one time, it just worked. I'm not confident at all on what actually solved the problem.
